acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/signing/ReplayProtectionAcceptanceTest.java

```diff
@@ -88,7 +90,7 @@ private void setUp(final String genesis) {
   public void wrongChainId() throws IOException {
     setUp("eth_hash_4404.json");
 
-    final JsonRpcErrorResponse error =
+    final SignerResponse<JsonRpcErrorResponse> signerResponse =
```

Comment:
would life be easier if  SignerResponse work was in a different PR?

Reply:
don't think it would

Reply:
The changes are quite small compared to rest of the PR and they required to the AT tests. So think would be easier for me at least to leave them in.